### ISSUE TITLE
Wrong order of elements for accessibility in chooser items

### ISSUE THREAD

**Issue by dusek** [textmate/bugs#10]
E.g. in File chooser, for an item, it first reads its path, only then the filename is read (VoiceOver user needs to first hear the filename as that is what will in 95% decide that it is not the candidate they were looking for, and needs further details as the path etc. only on closer inspection).

Same with bundle item chooser items - what is read is shortcut, title, then bundle. It should be first title, and only then bundle and shortcut (or shortcut and bundle).
